server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/CassandraBlobsDAOTest.java

```diff
@@ -77,4 +79,11 @@ void readBytesShouldReturnSplitSavedDataByChunk() {
         assertThat(new String(bytes, StandardCharsets.UTF_8)).isEqualTo(longString);
     }
 
+    @Test
+    void blobStoreShouldSupport100MBBlob() {
+        BlobId blobId = testee.save(new ZeroedInputStream(100_000_000)).join();
+        InputStream bytes = testee.read(blobId);
+        assertThat(bytes).hasSameContentAs(new ZeroedInputStream(100_000_000));
```

Comment:
Duplicating 100MB is probably costly

Reply:
It's not a duplication because it's an Inputstream that never materialize its content